CODE/explain_human_resps.py

```diff
@@ -85,11 +83,6 @@ def run_tfpr(tf_feat_mtx_dict, nontf_feat_mtx, features, label_df_dict, output_d
         logger.info('==> Cross validating response prediction model <==')
         tfpr_explainer.cross_validate(permute=permutations)
 
-        # TODO: to be removed in release
-        if not disable_shap:
-            logger.info('==> Analyzing feature contributions <==')
-            tfpr_explainer.explain()
-
```

Comment:
Deleting this entire section will remove SHAP value calculations completely, regardless of permutation or not.
Two ways to go about this (depending on how flag design):

1. Default to calculating SHAP values for regular runs, but not for permutation runs.
```suggestion
        if (not permutation) or (permutation and enable_permutation_shap):
            logger.info('==> Analyzing feature contributions <==')
            tfpr_explainer.explain()
```
2. Default to not calculating SHAP values regardless of runt type.

```suggestion
        if enable_shap:
            logger.info('==> Analyzing feature contributions <==')
            tfpr_explainer.explain()
```